extensions/git/package.json

```diff
@@ -1681,17 +1681,17 @@
         {
           "command": "git.stageSelectedRanges",
           "group": "2_git@1",
-          "when": "isInDiffRightEditor && !isInEmbeddedDiffEditor && config.git.enabled && !git.missing && gitOpenRepositoryCount != 0 && isInDiffEditor && resourceScheme =~ /^git$|^file$/"
+          "when": "isInDiffRightEditor && !isEmbeddedDiffEditor && config.git.enabled && !git.missing && gitOpenRepositoryCount != 0 && isInDiffEditor && resourceScheme =~ /^git$|^file$/"
```

Comment:
The change to the name makes internal naming with itself inconsistent (both is and isIn is used) as well as other context keys (`isInDiffRightEditor`)

Reply:
@Tyriar I was thinking about using `isInEmbeddedEditor` but that name was currently used by context key `isInWalkThroughSnippet`. It would require some consolidation (or getting confirmation whether the name or the context key id is misused). Also the name before was wrong and didn't reflect what it actually represents. Thus I went ahead with `isEmbeddedDiffEditor` first to unblock other tasks.